data/presets/shop/stationery.json

```diff
@@ -11,5 +11,8 @@
     "tags": {
         "shop": "stationery"
     },
-    "name": "Stationery Store"
+    "name": "Stationery Store",
+    "aliases": [
+        "Office Supply Shop"
```

Comment:
"Office supply store" would be more typically en_US

Reply:
more typical than "Stationery store"?

Reply:
I would’ve thought so for sure. I had never heard of it until joining OSM and hearing this Britishism. But [apparently “stationery store” is just as common](https://books.google.com/ngrams/graph?content=stationery+store%2Coffice+supply+store%2Coffice+store%2Coffice+supply+shop%2Coffice+shop&year_start=1800&year_end=2019&corpus=28&smoothing=3&direct_url=t1%3B%2Cstationery%20store%3B%2Cc0%3B.t1%3B%2Coffice%20supply%20store%3B%2Cc0%3B.t1%3B%2Coffice%20store%3B%2Cc0%3B.t1%3B%2Coffice%20supply%20shop%3B%2Cc0%3B.t1%3B%2Coffice%20shop%3B%2Cc0) in American English. Must be a regional thing?